src/Api/Models/Response/AttachmentResponseModel.cs

```diff
@@ -27,16 +26,15 @@ public AttachmentResponseModel(AttachmentResponseData data) : base("attachment")
         Url = $"{globalSettings.Attachment.BaseUrl}/{cipher.Id}/{id}";
         FileName = data.FileName;
         Key = data.Key;
-        Size = data.Size;
+        Size = data.Size.ToString();
         SizeName = CoreHelpers.ReadableBytesSize(data.Size);
     }
 
     public string Id { get; set; }
     public string Url { get; set; }
     public string FileName { get; set; }
     public string Key { get; set; }
-    [JsonNumberHandling(JsonNumberHandling.AllowReadingFromString | JsonNumberHandling.WriteAsString)]
```

Comment:
? why?

Reply:
Better question is, why is a numeric value a string in the first place? The automated rust mapping couldn't parse the string result properly.